docs/content/sre/tenant-operator/hibernation.md

```diff
@@ -15,30 +14,55 @@ hibernation:
 
 > Note: both sleep and wake schedules must be specified for your Hibernation schedule to be valid.
 
-Additionally, adding the following annotation `hibernation.stakater.com/exclude: 'true'` to a namespace, excludes that namespace from hibernating.
+Additionally, adding the `hibernation.stakater.com/exclude: 'true'` annotation to a namespace, excludes that namespace from hibernating.
 > Note that this won't wake up an already sleeping namespace before the wake schedule.
 
 ## Resource Supervisor
 
-Adding a Hibernation Schedule to a Tenant creates an accompanying ResourceSupervisor custom resource.
+Adding a Hibernation Schedule to a Tenant creates an accompanying ResourceSupervisor Custom Resource.
 The Resource Supervisor stores the Hibernation schedules and manages the current and previous states of all the applications, whether they are sleeping or awake.
 
-When the sleep timer is activated, the controller for the resource stores the details of your applications; including the number of replicas, configurations, etc., in the namespaces owned by the tenant and will then put your applications to sleep. When the wake timer is activated, the controller wakes up the applications using their stored details.
+When the sleep timer is activated, the Resource Supervisor controller stores the details of your applications (including the number of replicas, configurations, etc.) in the applications' namespaces and then puts your applications to sleep. When the wake timer is activated, the controller wakes up the applications using their stored details.
 
-Enabling ArgoCD support for Tenants will also hibernate applications in the tenants `appProjects`.
+Enabling ArgoCD support for Tenants will also hibernate applications in the tenants' `appProjects`.
 
 ```yaml
 apiVersion: tenantoperator.stakater.com/v1beta1
 kind: ResourceSupervisor
 metadata:
- name: sigma
+  name: sigma
 spec:
- argocd:
-   appProjects:
-     - sigma
-   namespace: openshift-gitops
- hibernation:
-   sleepSchedule: 42 * * * *
-   wakeSchedule: 45 * * * *
+  argocd:
+    appProjects:
+      - sigma
+    namespace: openshift-gitops
+  hibernation:
+    sleepSchedule: 42 * * * *
+    wakeSchedule: 45 * * * *
+  namespaces:
+    - tenant-ns1
+    - tenant-ns2
 ```
-> Currently, Hibernation is available for only StatefulSets and Deployments
+> Currently, Hibernation is available only for StatefulSets and Deployments.
+
+It is also possible to hibernate non-tenant namespaces by creating a ResourceSupervisor CR manually. This ResourceSupervisor definition will contain the hibernation cron schedule, the names of the namespaces to be hibernated, and the names of the ArgoCD AppProjects whose ArgoCD Applications have to be hibernated (as per the given schedule).
```

Comment:
it would be better to define this feature as something that can work across tenant namespaces